crud-app/app/Http/Controllers/CrudUserController.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+use Illuminate\Validation\Rules;
+
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Auth\Events\Registered;
+
+
+use \App\Models\User;
+
+class CrudUserController extends Controller
```

Comment:
By convention, this should be called `UserController`. No need to change now but wanted you to know.

Reply:
I updated the code and pushed the changes after the video and user update works correctly now.

Reply:
rerequest me as a reviewer on this PR if you are ready